conf.py

```diff
@@ -100,7 +100,7 @@
 # default_role = 'cpp:any'
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "xcode"
+pygments_style = "solarized-dark"
```

Comment:
I think this makes the text too light in light mode

Reply:
That was the only one I found that more or less works for both light and dark... I see Sphinx supports `pygments_dark_style` but not as a user setting, so I'll look at creating a custom theme that extends `alabaster` so we can use `xcode` for light and `solarized-dark` (or I prefer github-dark)